### ISSUE TITLE
Some Push-to-talk hotkeys aren't assigning correctly in the new Steam Chat

### ISSUE THREAD

**Issue by Rattlewrench** [ValveSoftware/steam-for-linux#5523]
#### Your system information

* Steam client version (build number or date): Jun 12 2018
* Distribution (e.g. Ubuntu): Manjaro Linux (64-bit)
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
I'm unable to assign _Mouse 4_ or _Mouse 5_ as the push-to-talk button in the **Friends List Settings**.

Unlike buttons like _Shift_, _Alt_, _Tab_ and other, which aren't possible to assign ("_None_" is shown when I try to assign them), _Mouse 4_ and _5_ buttons doesn't seem to be registered at all - the blue frame keeps flashing and text "_Press key or mouse-button to assign_" is still shown.

Also, a lot of buttons have a problem of assigning another button instead of itself. And the button shown in the settings (not the one I tried to assign) will be the one that actually is working as PTT.
Here's the list of buttons and what they are turning into in the settings:
```
Num 0 - `
Num 1 - A
Num 2 - B
Num 3 - C
Num 4 - D
Num 5 - E
Num 6 - F
Num 7 - G
Num 8 - H
Num 9 - I
Num * - J
Num + - K
Num - - M
Num . - N
Num / - O
F1 - P
F2 - Q
F3 - R
F4 - S
F5 - T
F6 - U
F7 - V
F8 - W
F9 - X
F10 - Y
F11 - Z
F12 - [
Left Super (Win) - [
Right Super (Win) - \
INS - -
DEL - .
PG DOWN - '
PG UP - 1
END - 3
HOME - 4
Left Arrow - 5
Up Arrow - 7
Down Arrow - 9
Right Arrow - '
```

#### Steps for reproducing this issue:

1. Go to the "Friend List Settings",
2. Go to the "Voice" tab,
3. In the "PUSH-TO-TALK" section, enable push-to-talk if it wasn't already,
4. Click on the "Push-to-talk hotkey assigned as" input field,
5. Press **F1** _(or any key from the list above)_
6. Instead of **F1** hotkey will be assigned to the **P**

**Comment by Plagman** [comment]
Can you re-test with today's Beta update?

**Comment by Rattlewrench** [comment]
I have tested as much buttons as I could think of - most of them are working perfectly. Great stuff!
Few minor issues:
**App** key can be assigned, but doesn't do anything when it is.
After assigning any **Alt** key (right or left) the left one will be working as PTT.
Same with the **Ctrl** keys - only the left one will be working, whether I have assigned right or left one.
If I'll assign **Num Enter** as push-to-talk, the regular **Enter** will do that function, instead.

I don't know if any of that was intended behaviour.
But both left and right **Win** buttons can be assigned as the push-to-talk, so the logic didn't look entirely consistent.
